playwright/page-objects/cluster-roles-access-page.ts

```
@@ -0,0 +1,136 @@
+import { Page, Locator, expect } from '@playwright/test';
+import { BasePage } from './base-page';
```

ESLint

[error] Run autofix to sort these imports!

(simple-import-sort/imports)